test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -504,8 +503,8 @@ class test_ucp_sockaddr : public ucp_test {
             recv_req = recv(to, &recv_data, sizeof(recv_data),
                             rstream_complete_cbx, NULL);
         } else {
-            UCS_TEST_ABORT("unsupported communication type "
-                    << send_recv_type);
+            UCS_TEST_ABORT("unsupported communication type " +
+                           ucs::to_string(send_recv_type));
```

Comment:
1) how does it work without casting `const char*` to `std::string`?
2) I guess `ucs::to_string` is not needed anymore since we use C++11, IMO better to use `std::to_string`

Reply:
> 1. how does it work without casting `const char*` to `std::string`?

I think it selects this `+` operator's implementation: 
```
template< class CharT, class Traits, class Alloc >
    std::basic_string<CharT,Traits,Alloc>
        operator+( const CharT* lhs,
                   const std::basic_string<CharT,Traits,Alloc>& rhs );
````
https://en.cppreference.com/w/cpp/string/basic_string/operator%2B

> 2. I guess `ucs::to_string` is not needed anymore since we use C++11, IMO better to use `std::to_string`

fixed

Reply:
also, I found other places in our gtest where it is used in the same way